radio/src/gui/colorlcd/layouts/layout2x4.cpp

```diff
@@ -79,6 +79,27 @@ class Layout2x4: public Layout
 
       return zone;
     }
+
+    void paint(BitmapBuffer * dc) override;
 };
 
+void Layout2x4::paint(BitmapBuffer* dc)
+{
+  rect_t fullScreen = Layout::getMainZone();
+  fullScreen.w /= 2;
+  if (getOptionValue(OPTION_PANEL1_BACKGROUND)->boolValue) {
+    lcdSetColor(getOptionValue(OPTION_PANEL1_COLOR)->unsignedValue);
+    dc->drawSolidFilledRect(fullScreen.x, fullScreen.y, fullScreen.w,
+                            fullScreen.h, CUSTOM_COLOR);
+  }
+
+  if (getOptionValue(OPTION_PANEL2_BACKGROUND)->boolValue) {
+    fullScreen.x += fullScreen.w;
+    lcdSetColor(getOptionValue(OPTION_PANEL2_COLOR)->unsignedValue);
+    dc->drawSolidFilledRect(fullScreen.x, fullScreen.y, fullScreen.w,
+                            fullScreen.h, CUSTOM_COLOR);
+  }
+  Layout::paint(dc);
```

Comment:
This one currently empty, so it does not matter, but normally you would want to have it used first, as it might draw a background or something like that.